docker/py-vnc/Dockerfile

```diff
@@ -0,0 +1,52 @@
+ARG BASE_IMAGE=renku/renkulab-py:latest
+FROM ${BASE_IMAGE}
+
+LABEL maintainer="Swiss Data Science Center <info@datascience.ch>"
+
+USER root
+
+RUN apt-get update \
+    && apt-get install -yq --no-install-recommends \
+        dbus-x11 \
+        firefox \
+        net-tools \
+        less \
+        xfce4 \
+        xfce4-panel \
+        xfce4-session \
+        xfce4-settings \
+        xorg \
+        xubuntu-icon-theme \
+        xterm \
+    && apt-get autoremove --purge \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -rf /tmp/* \
+    && find /var/log -type f -exec cp /dev/null \{\} \;
+
+# Get noVNC
+RUN cd /opt && \
+    curl -sSfL https://github.com/novnc/noVNC/archive/v1.1.0.tar.gz | tar -zxf -
```

Comment:
could we parametrize the version here so that it can be reused later on when patching the config file? just to avoid errors in the future.

Reply:
yes, exactly

Reply:
Good idea. I'll make those changes